bazel/grpc_grpc.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/BUILD b/BUILD
```

Comment:
why do you need this patch? Isn't `CARES_STATICLIB` defined in the cc_library `ares` so it should propagate?

Reply:
There's a small misunderstanding here.

Using the static libcrt.lib and avoiding the linkage to msvcrt.dll has nothing to do at all with whether a given library is a static (non-PIC) source compilation. This patch alters the Microsoft CRT we bind to, as opposed to the object files we are compiling.